### DISCUSSION TITLE
How can I leave the Windows Package Manager Insiders Program?

### DISCUSSION THREAD

**Discussion by furcanomer** [microsoft/winget-cli#1164] [Q&A] [answered]
How can I leave the Windows Package Manager Insiders Program?

I joined the program here. ------>>> https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR-NSOqDz219PqoOqk5qxQEZUMVVCT1IwVEpLSklZS0dDRFZEUjZUOU9ZWi4u

**Comment by Masamune3210** [comment]
Just don't use the extra features, pretty sure all the insider track for this affects is winget and the Package Installer

**Comment by denelon** [comment, marked as answer]
You can opt-out of the Windows Package Manager Insider program by filling out our [form](https://aka.ms/AppInstaller_Unsubscribe). 
>Note: The removal process is not 100% automated. There may be a delay of one or two business days.

**Reply by furcanomer** [reply]
Thank you.

**Reply by furcanomer** [reply]
The beta version is currently installed. Do I need to do anything extra after exiting the Insider program?

**Reply by denelon** [reply]
@furcanomer if you wish to return to the current version for the "App Installer", you will need to uninstall the "insider" version on your system, and then install it from the Microsoft Store again. I'd suggest waiting a few days after the removal request so we have enough time to process.

If you do nothing, when the next stable release is published with a higher version than what you have currently installed, you will then get that version from the Microsoft Store automatically (assuming no Group Policy prevents that behavior).

**Reply by adrianghc** [reply]
@denelon I have used the form but despite four business days having passed, I'm still in the Insider program. Is this still normal or could something have slipped?

**Reply by denelon** [reply]
@adrianghc 

You would need to install the "stable" version or wait until stable surpasses the version you have installed. If you install the stable version, and the store still upgrades you to the preview version, let me know.

**Reply by adrianghc** [reply]
> @adrianghc
> 
> You would need to install the "stable" version or wait until stable surpasses the version you have installed. If you install the stable version, and the store still upgrades you to the preview version, let me know.

@denelon I have installed the "stable" version but keep being updated to the preview version through the Store, which is how I know that I'm still in Insiders.

**Reply by denelon** [reply]
Please send an e-mail to winget-feedback@microsoft.com and let us know which MSA you want us to check on. We can make sure it gets removed.

**Reply by adrianghc** [reply]
> Please send an e-mail to [winget-feedback@microsoft.com](mailto:winget-feedback@microsoft.com) and let us know which MSA you want us to check on. We can make sure it gets removed.

Thank you, I've just sent an email.

**Reply by denelon** [reply]
You will also get a preview version on any Windows Insider Build.

**Reply by adrianghc** [reply]
> You will also get a preview version on any Windows Insider Build.

Yes, I know, but I'm on the stable version of Windows 11.

**Reply by VincentJoshuaET** [reply]
I filled the form more than a week ago already, then reinstalled the stable version just earlier. I checked for updates in Microsoft Store and got the preview version of winget again. I already sent an email as well.

**Reply by KevinLaMS** [reply]
Sorry   VincentJoshuaET
This is not an automated process.
I processed all removal requests this AM.  If you want to email me directly, I can double check that account has been removed.
Thanks
kevin (kevinla@microsoft.com)
